packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.tsx

```diff
@@ -79,6 +79,14 @@ export class GroupedList extends BaseComponent<IGroupedListProps, IGroupedListSt
     }
   }
 
+  public componentWillMount() {
+    const { groupProps } = this.props;
+
+    if (groupProps && groupProps.initialAllGroupsCollapsed) {
+      this.toggleCollapseAll(groupProps.initialAllGroupsCollapsed);
```

Comment:
This would trigger a forceUpdate and rerender...

Reply:
good point, maybe extract the logic to iterate over the groups and set `isCollapsed` in the constructor when we create the initial state? 

That would essentially modify `props`, but that's already a problem with the current code

Reply:
Ah, you're right about that; not ideal. I also just realized I can set `isCollapsed: true` on any/all of my groups and they will be collapsed upon render, so I don't think I need this PR anymore